apps/src/music/Controls.jsx

```diff
@@ -57,26 +58,20 @@ const Controls = ({
   return (
     <div id="controls" className={moduleStyles.controlsContainer}>
       {isShowingBeatPad && renderBeatPad()}
-      {/*Placeholder button area, currently hidden*/}
       <div
-        className={classNames(
-          moduleStyles.controlButtons,
-          moduleStyles.side,
-          moduleStyles.hide
-        )}
+        className={classNames(moduleStyles.controlButtons, moduleStyles.side)}
       >
         <FontAwesome
-          icon={'th'}
-          style={{fontSize: 30}}
+          icon={'info-circle'}
           className={moduleStyles.iconButton}
+          onClick={toggleInstructions}
```

Comment:
Should this icon move to the right when the instructions are on the right?

Reply:
Hmm, yeah good point. I'll update this.